torch/csrc/distributed/rpc/utils.h

```diff
@@ -14,7 +14,8 @@ TORCH_API std::unique_ptr<RpcCommandBase> deserializeRequest(
 // Given an RPC message received as a response over the wire, deserialize it
 // into the appropriate 'RpcCommandBase' type.
 TORCH_API std::unique_ptr<RpcCommandBase> deserializeResponse(
-    const Message& response);
+    const Message& response,
+    MessageType& wrappedMsgType);
```

Comment:
nit: Can we enhance the docs to mention what `wrappedMsgType` refers to? Also, can we make `wrappedMsgType` be an optional argument and default to `UNKNOWN`? Then in the `FORWARD_AUTOGRAD_RESP` case we assert that it is not unknown? (since its used only there).

Reply:
will add doc. and let me try to change wrappedMsgType default to UNKNOWN

Reply:
hmm, wrappedMsgType is non const reference, it can not be bind to temp default value, so I'm keeping this as it is right now